[2022-11-18 16:09:47,353]: Skip avahi_enable search...
[2022-11-18 16:09:47,353]: SSL is not enabled...
[2022-11-18 16:09:47,355]: [-> CONNECT           ] Connect to Host: 127.0.0.1:11083
[2022-11-18 16:09:47,358]: Some error
[2022-11-18 16:09:47,359]: Traceback message : Traceback (most recent call last):
  File "/home/jyeom/anaconda3/envs/py37tf26/lib/python3.7/site-packages/big_thing_py/manager_thing.py", line 157, in run
    raise Exception('_g_exit was set!!!')
Exception: _g_exit was set!!!

Error: _g_exit was set!!!
[2022-11-18 16:09:47,359]: Traceback message : Traceback (most recent call last):
  File "/home/jyeom/anaconda3/envs/py37tf26/lib/python3.7/site-packages/big_thing_py/manager_thing.py", line 228, in receive_staff_message_thread_func
    self._handle_staff_message(msg)
  File "/home/jyeom/big-thing-py-examples/manager_thing/lora_manager_thing/lora_manager_thing.py", line 36, in _handle_staff_message
    for idx, lora_device in lora_device_list.items():
AttributeError: 'list' object has no attribute 'items'

Error: 'list' object has no attribute 'items'
[2022-11-18 16:09:49,362]: Traceback message : Traceback (most recent call last):
  File "/home/jyeom/anaconda3/envs/py37tf26/lib/python3.7/site-packages/big_thing_py/manager_thing.py", line 183, in wrapup
    thread.exit()
AttributeError: 'SoPThread' object has no attribute 'exit'

Error: 'SoPThread' object has no attribute 'exit'
